src/licensedcode/data/rules/agpl-3.0-plus_or_commercial-license_4.RULE

```diff
@@ -1,5 +1,5 @@
 ---
-license_expression: agpl-3.0-plus OR commercial-license
```

Comment:
While the change is correct note that the rule filenames reflect the license-expression of the rule and a change in the license-expression needs to happen like this:

1. Deprecate the current rule using the `is_deprecated` attribute
2. Reindex the licenses with `scancode-reindex-licenses` (This will remove the deprecated rule from the index so you can add a new one, otherwise two rules with the same text cannot exist in the index)
3. Add a new rule for just `agpl-3.0-plus` and add notes about the origin, that this was previously the src/`agpl-3.0-plus_or_commercial-license_4.RULE` rule. Add the issue reference in the commit. (You can paste the rule content at `etc/scripts/licenses/buildrules-template.txt` and then run `python etc/scripts/licenses/buildrules.py etc/scripts/licenses/buildrules-template.txt` in an activated virtualenv, and this will create the rule automatically, with the appropriate filename)